web_responsive/static/src/js/web_responsive.js

```diff
@@ -358,7 +367,24 @@ odoo.define('web_responsive', function (require) {
             } else {
                 this._super.apply(this, arguments);
             }
-        }
+        },
+    });
+
+    // Hide AppMenu & remove waiting anim when loaded action
+    ActionManager.include({
+        doAction: function () {
+            return this._super.apply(this, arguments).then(function () {
+                var $app_menu = $('.o_menu_apps .dropdown');
+                if ($app_menu.length) {
+                    if ($app_menu.hasClass('show')) {
+                        $app_menu.dropdown('toggle');
+                    }
+
+                    $app_menu.find('img.o-app-icon-waiting').removeClass(
```

Comment:
As always, the less strict you are, the better :blush::

```suggestion
                    $app_menu.find('.o-app-icon-waiting').removeClass(
```